plugins/wazuh-fleet/public/components/agents/details/details.tsx

```diff
@@ -8,68 +8,49 @@ import {
   EuiContextMenuPanel,
   EuiContextMenuItem,
   EuiHorizontalRule,
-  EuiTabbedContent,
+  // EuiTabbedContent,
   EuiLoadingContent,
   EuiContextMenu,
-  EuiIcon,
 } from '@elastic/eui';
-import { Agent } from '../../../../common/types';
+import { IAgentResponse } from '../../../../common/types';
+import { getAgentManagement } from '../../../plugin-services';
 import { AgentResume } from './resume';
-import { AgentDashboard } from './dashboard';
-import { AgentNetworks } from './networks';
+// import { AgentDashboard } from './dashboard';
+// import { AgentNetworks } from './networks';
 
 export interface AgentDetailsProps {
-  useDataSource: any;
-  FleetDataSource: any;
-  FleetDataSourceRepository: any;
+  indexPatterns: any;
+  filters: any[];
```

Comment:
The filters prop is typed as any. Using any defeats the purpose of TypeScript. Consider defining a specific type for filters to improve type safety and maintainability.

import { Filter } from '../../../../src/plugins/data/public';

```suggestion
  filters: Filter[];
```